### ISSUE TITLE
Hint text not behaving correctly

### ISSUE THREAD

**Issue by drew-wallace** [NoHomey/material-ui-number-input#8]
If you have a completely blank field with strategy set to ignore, the hint text is shown. Then, type a letter. The hint text is gone, yet there is still nothing in the box.

**Comment by NoHomey** [comment]
Yes because strategy ignore means it will ignore all incorrect values. The hint text is controlled by `material-ui` itself, if you unfocus the input it should appear since the input is empty.
